app/javascript/mastodon/components/dropdown_menu.js

```diff
@@ -1,53 +1,50 @@
 import React from 'react';
 import ImmutablePropTypes from 'react-immutable-proptypes';
-import Dropdown, { DropdownTrigger, DropdownContent } from 'react-simple-dropdown';
 import PropTypes from 'prop-types';
+import IconButton from './icon_button';
+import { Overlay } from 'react-overlays';
+import { Motion, spring } from 'react-motion';
 
-export default class DropdownMenu extends React.PureComponent {
+class DropdownMenu extends React.PureComponent {
 
   static contextTypes = {
     router: PropTypes.object,
   };
 
   static propTypes = {
-    isUserTouching: PropTypes.func,
-    isModalOpen: PropTypes.bool.isRequired,
-    onModalOpen: PropTypes.func,
-    onModalClose: PropTypes.func,
-    icon: PropTypes.string.isRequired,
     items: PropTypes.array.isRequired,
-    size: PropTypes.number.isRequired,
-    direction: PropTypes.string,
-    status: ImmutablePropTypes.map,
-    ariaLabel: PropTypes.string,
-    disabled: PropTypes.bool,
+    onClose: PropTypes.func.isRequired,
+    style: PropTypes.object.isRequired,
+    placement: PropTypes.string.isRequired,
+    arrowOffsetLeft: PropTypes.number.isRequired,
+    arrowOffsetTop: PropTypes.number.isRequired,
```

Comment:
Is `.isRequired` unnecessary?

ref https://github.com/tootsuite/mastodon/pull/5018/files#r139862589